# redirect hoạt động bằng cách ném ra một exception nếu trong server action ta để redirect trong khối try thì nó sẽ bị khối catch bắt và không thể thực hiện như mong muốn
